docs/DeveloperGuide.md

```diff
@@ -1019,6 +1019,78 @@ starting point for testers to work on; testers are expected to do more *explorat
     2. Re-launch the app by double-clicking the jar file. Check the command history again with `history`. <br>
        Expected: The same command history (possibly with one extra `history` and/or `exit` command) is displayed.
 
+### Issue
+
+1. Adding an issue
+
+    1. Prerequisites: Room `01-234` exists in SunRez.
+
+    2. Test case: `iadd r/01-234 d/Collapsed ceiling` <br>
+       Expected: A feedback message in the result box indicating the issue was successfully added. The issue should be visible in the issue panel.
+
+    3. Test case: `iadd r/01-234 d/Collapsed ceiling t/3000/01/01 12:00pm` or any timestamp in the future <br>
+       Expected: An error message indicating that timestamps should not be in the future.
+
+    4. Test case:
+        * `iadd r/01-234 d/Collapsed ceiling t/2020/01/01 12:00pm s/pending c/Ceiling g/a` <br>
+        * `iadd r/01-234 d/Collapsed ceiling t/2020/01/01 12:00pm s/pending c/Ceiling g/b`
+       Expected: An error message indicating that the issue already exists in SunRez as the issue to be added is exactly the same excluding tags.
+
+3. Finding issues
+
+    1. Prerequisites: Issues belonging to room `01-234` and `03-325` with descriptions `Broken table` and `Door broke in half` respectively
+
+    2. Test case: `ifind 01`
+       Expected: Issue panel should be filtered to show all rooms with `01` in their room number, description, or tag. This includes the `Broken table` issue for `01-234`.
+
+    3. Test case: `ifind broke`
+       Expected: Issue panel should be filtered to show all rooms with `broke` (case insensitive) in their room number, description, or tag. This include both issues stated in the prerequisites.
```

Comment:
Minor grammar nit:

>This include both issues...

Suggestion:

> This includes both issues...


Saw this in a few points below also.